src/components/AreaChart.tsx

```diff
@@ -30,23 +31,29 @@ type IAreaChartNonBucketProps = Subtract<IAreaChartProps, IAreaChartBucketProps>
  * is a component with bucket props measures, viewBy, stacksBy, filters
  */
 export function AreaChart(props: IAreaChartProps): JSX.Element {
+    showWarningForAreaChart(props);
```

Comment:
`showWarningForAreaChart` is too generic name... only it tells me that it is going to show warning for area chart which is well not true in all cases :) 

rather than that consider to rename it to something more specific something like

`verifyBuckets / validateBuckets` which is always checking buckets and what is coming to this component.